tests/apps/actordotnet/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/aspnet:3.1 AS base
+WORKDIR /app
+EXPOSE 3000

```

Comment:
likely doesn't matter, but asp.net core will listen on 80 in a container by default, not 3000

Reply:
Our e2e tests expect 3000 on test apps. I decided to keep them as expected by the test.

Reply:
ok